src/powershell/install.sh

```diff
@@ -162,6 +163,12 @@ if [ ${#POWERSHELL_MODULES[@]} -gt 0 ]; then
     done
 fi
 
+# If URL for powershell profile is provided, download it to '/opt/microsoft/powershell/7/profile.ps1'
+if [ -n "$POWERSHELL_PROFILE_URL" ]; then
```

Comment:
Can we add test scenarios to validate these PR changes? Feel free to provide me a script, and I can host it publicly which will be accessible to the test scenario. This is good for maintainability and that way you don't have to maintain/support it long term.

Reply:
Yep for sure, I do not have a public profile atm. For something to test you can use https://github.com/ChrisTitusTech/powershell-profile , the value you will need to use for this parameter is `https://github.com/ChrisTitusTech/powershell-profile/raw/main/setup.ps1`
There are lot more profiles [here](https://github.com/topics/powershell-profile) if you want additional tests.

Reply:
Thanks, I created a fork at https://github.com/codspace/powershell-profile/tree/main

Can you write tests by using that? Would be great to validate these changes and have some tests added which massages this new Feature option.

Reply:
Hey @samruddhikhandale,
Apologies for delay in getting back to you, I was bit busy.
Picking this up again, I see three files [here](https://github.com/devcontainers/features/tree/main/test/powershell), but not really clear on how those are being used. Can you please share any docs on how to write test for this or you can guide me through this?

Reply:
NM, I found the test docs. Will get the test implemented in next day or two.

Reply:
Alright, I've added test now. I create a simple profile [here](https://github.com/nebula-it/features/blob/patch-2/test/powershell/testProfile.ps1) for testing, as the one I linked is too extensive and not really a good option for testing.

With the current test, if the variable `$env:ProfileLoaded` exists and returns value `True`, that means our profile has been successfully loaded.